reference/forms/types/entity.rst

```diff
@@ -202,9 +202,9 @@ query_builder
 Allows you to create a custom query for your choices. See
 :ref:`ref-form-entity-query-builder` for an example.
 
-The value of this option can either be a ``QueryBuilder`` object or a Closure.
+The value of this option can either be a ``QueryBuilder`` object, a Closure or null.
```

Comment:
@WouterJ see https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php#L167 and https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bridge/Doctrine/Form/Type/EntityType.php#L43 but this change should be done in 2.3 though